CHANGELOG.md

```diff
@@ -1,9 +1,16 @@
 # Changelog
 
-## Unreleased
-
+## [v0.8.2] 2023-12-06
 * Bug fixes:
-    * Follow Amazon EC2 IMDSv2 requirements to set `running-in-aws`
+    * Enable AWS check with IMDSv2
+    * Add region to running in AWS check
+    * Handle opening multi-file granules
```

Comment:
I didn't get a chance to look at this, but does it add any new surface to the API that we should call out by name, e.g. new method?